### ISSUE TITLE
Unable to copy to clipboard on Ubuntu

### ISSUE THREAD

**Issue by gkotian** [asweigart/pyperclip#112]
Trying to do `pyperclip.copy()` on my Ubuntu 17.04 (zesty) machine results in the following error:
```
** (test.py:24541): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-mS4n3lk0Uf: Connection refused
```

I installed pyperclip via:
```
$> pip install pyperclip 
Collecting pyperclip
Installing collected packages: pyperclip
Successfully installed pyperclip-1.6.0
```

**Comment by gkotian** [comment]
My `test.py` file is simply:
```python
#!/usr/bin/python

import pyperclip
pyperclip.copy('The text to be copied to the clipboard.')
```

**Comment by gkotian** [comment]
As a workaround, I'm currently using:
```python
#!/usr/bin/python

import os
text = 'The text to be copied to the clipboard.'
os.system("echo '{}' | xclip -selection clipboard".format(text))
```

and this successfully puts the text onto the clipboard.

**Comment by gkotian** [comment]
Interestingly, in the python REPL, the `Couldn't connect to accessibility bus` warning is shown, but `pyperclip.paste()` does paste the correct text.
```
$> cat test.py
#!/usr/bin/python

import pyperclip

pyperclip.copy('The text to be copied to the clipboard.')
pyperclip.paste()
$>
$>
$> ./test.py 

** (test.py:25750): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-mS4n3lk0Uf: Connection refused
$>
$>
$> python
Python 2.7.13 (default, Jan 19 2017, 14:48:08) 
[GCC 6.3.0 20170118] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyperclip
>>> pyperclip.copy('The text to be copied to the clipboard.')

** (.:25761): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-mS4n3lk0Uf: Connection refused
>>> pyperclip.paste()
'The text to be copied to the clipboard.'
>>> 
$>
```

**Comment by tannishpage** [comment]
To solve this issue install xclip `sudo apt-get install xclip` that solved it for me. If that also dosen't work try installing xsel `sudo apt-get install xsel`

**Comment by dileepamax** [comment]
installing xclip `sudo apt-get install xclip` and run your script

**Comment by Lu251970** [comment]
Hi !. The solution you suggested was successfully used in both codes and worked as expected. However, I would like to inquire whether it is feasible to have xclip copy the text from the clipboard instead of storing it in a variable, and then perform the modifications and resend it back to the clipboard. Thank you! Luciana.
